torch_frame/nn/models/bcauss.py

```diff
@@ -0,0 +1,138 @@
+from typing import Any, Tuple
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+from torch.nn import Linear, Module, ReLU, Sequential
+
+import torch_frame
+from torch_frame import TensorFrame, categorical, numerical
+from torch_frame.data.stats import StatType
+
+
+class MLPBlock(Module):
```

Comment:
I thought we have an `MLP` implementation. Can't we use it?

Reply:
That one is a model and it involves specifying an encoder. I don't think the `BCAUSS` mode uses any encoder.